src/Analysis/Engine/Impl/DependencyResolution/IAvailableImports.cs

```diff
@@ -17,5 +17,6 @@
 namespace Microsoft.PythonTools.Analysis.DependencyResolution {
     internal interface IAvailableImports {
         string Name { get; }
```

Comment:
Comment would be helpful what does `name` means. Interface name implies is plural but data returned seem to imply a single entity

Reply:
Yeah, I actually think I need better name for the interface as well.

Reply:
Renamed